### DISCUSSION TITLE
Is it possible to add a nhentai downloader?

### DISCUSSION THREAD

**Discussion by mhdy2233** [Difegue/LANraragi#1114] [Q&A]
Is it possible to add a nhentai downloader? I found an api for nhentai that might help in making a downloader.
Project address: [SylveonDeko/NHentaiAPI: A (full) nHentai API implementation for .](https://github.com/SylveonDeko/NHentaiAPI)

**Comment by Difegue** [comment]
No -- nH doesn't provide any sorts of download API, and support for downloaders in LRR is limited to links that return a single file. (as in a zip, rar or pdf)  

See https://github.com/Difegue/LANraragi/discussions/804 for more details.
